services/notify/src/api.js

```diff
@@ -31,6 +31,7 @@ builder.declare({
   ].join('\n'),
 }, async function(req, res) {
   debug(`Received request to send email to ${req.body.address}`);
+  this.monitor.email({address: req.body.address});
```

Comment:
Purely on the basis of readability, I'd prefer this to be something like `this.monitor.log.email(..)`.  Maybe it would read better with a past-tense name like `this.monitor.emailSent`?